parsons/action_network/action_network.py

```diff
@@ -321,21 +325,15 @@ def update_person(self, entry_id, background_processing=False, **kwargs):
         logger.info(f"Person {entry_id} successfully updated")
         return response
 
-    def get_tags(self, limit=None, per_page=25, page=None):
```

Comment:
We may want to add `**kwargs` to the method signature to cover backwards compatibility if anyone had previously declared `per_page` or `page` on this method. Removing them otherwise might cause an "unexpected keyword argument" TypeError.

Reply:
What do you think about this implementation? 22952c73aed29a729da3f3067654a9d9dfe6ab7b

If the page argument was used previously, it would have resulted in an Exception because it would have called a non-existent method. So I think we can be confident that getting rid of that argument will not break any code that wasn't already broken.

Reply:
To be explicit, I re-include the `per_page` argument and issue a DeprecationWarning if it is used.

I think this is more concise and clear than adding `**kwargs`, which is more open-ended and I think would introduce more ambiguity.